Reservia.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+
+<html lang="fr"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <title>Reservia</title>
+   
+    <link rel="stylesheet" href="./Reservia_files/style.css">
+ 
+
+</head>
+<body> 
+
+    <header class="pageHead">
+        <img class="pageHead__logo" src="./Reservia_files/reservia_logo.svg" alt="logo Reservia">
+        <nav class="pageHead__nav">
+            <a class="pageHead__navItem" href="hebergement.html">Hébergements</a>
+            <a class="pageHead__navItem" href="Activities.html">Activités</a>
+        </nav>
+        <a class="pageHead__itemRegister" href="page_register.html">S'inscrire</a>
+    </header>
+
+
+
+    <div id="PageRelease">
+        <h2>Trouvez votre hébergement pour des vacances de reves</h2>
+        <h5>En plein centre ville ou en pleine nature</h5>
+        <div id="garden">
+            <img class="fit-picture" src="Reservia_files/maps.png"alt="search">
+            <form action="/" method="post">
+                <!--<img class="fit-picture" src="Reservia_files/maps.png"alt="search">-->
+                <div class="label">   
+                    <input type="text" id="name" name="user_name" placeholder="Marseille,France">    
+                </div>
+                <div class="button">
+                    <button id="ida" type="submit">Recherche</button>
+                </div>
+            </form>
+        </div>
+        <br>
+        <span class="filter">Filtres</span>     
+        <button class="pictureButton" type=submit>
+            <img id="img1"src="Reservia_files/eco.png">
+            </button>
+            <button class="pictureButton" type=submit>
+                <img id="img2"src="Reservia_files/Familial.PNG">
+                </button>
+                <button class="pictureButton" type=submit>
+                    <img id="img3" src="Reservia_files/Romantic.png">
+                    </button>
+                    <button class="pictureButton" type=submit>
+                        <img id="img4" src="Reservia_files/Autorisedanimals.PNG">
+                        </button>
+      </div>
+      
+        <p ><img class="Informationpicture" src="Reservia_files/information.PNG"alt="search">
```

Comment:
`Informationpicture` doesn't match the same pattern as the one in header or in buttons just above. is it normal? for a better readability you should work with the same naming convention when ever you write some CSS code.

example of not the same patterns: 
```
<button class="pictureButton" type=submit>
<img class="Informationpicture" src="Reservia_files/information.PNG"alt="search">
<header class="pageHead">
```

you have to choose one and stick to it :)

a very good article to look at with a lot of informations on best practices: https://hackernoon.com/best-practice-in-css-organisation-and-naming-conventions-4d103ujy

Reply:
I changed InformationPicture to informationPicture in order to respect the naming conventions.